core/shared/src/test/scala/kantan/codecs/resource/ResourceIteratorTests.scala

```diff
@@ -321,12 +320,6 @@ class ResourceIteratorTests
     }
   }
 
-  test("toIterable should behave as expected") {
```

Comment:
Why is this test deleted? does it no longer pass?

Reply:
The method is deprecated

Reply:
Ah yes, `toIterable` is deprecated from 2.13 - but `to(Iterable)` is perfectly ok, apparently.

I can think of 3 ways around this that don't involve not testing code that used to be tested:
- using the collection compat library for tests, so that you can use the cross-version way of turning something into an `Iterable`.
- dropping support for `toIterable` in `ResourceIterator`, I _think_ it was merely there because I copied the `Iterator` API.
- using the same mechanism I employ to have different implementations of `ResourceIterator` methods in Scala 2.12 and 2.13 to have the 2.12 test use `toIterable`, the 2.13 one `to(Iterable)`.

Reply:
Fixed with compat library